caas/kubernetes/provider/k8s.go

```diff
@@ -16,9 +16,9 @@ import (
 	jujuclock "github.com/juju/clock"
 	"github.com/juju/collections/set"
 	"github.com/juju/errors"
+	"github.com/juju/juju/core/arch"
```

Comment:
Wrong import stanza. Didn't the linter or SA check these? This happened in many of the files.